services/gateway/shared/ready.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const BluebirdPromise = require('bluebird');
+
+const { MoleculerError } = require('moleculer').Errors;
+
+const getReady = async broker => {
+    const coreMethods = {
+        lisk_accounts: 'core.accounts',
+        lisk_blocks: 'core.blocks',
+        lisk_transactions: 'core.transactions',
+        // lisk_delegates: 'core.delegates',
+        // lisk_peers: 'core.peers',
+    };
+    const services = await BluebirdPromise.map(
+        Object.getOwnPropertyNames(coreMethods),
+        async key => {
+            const service = {};
+            const response = await broker.call(coreMethods[key]);
+            service[key] = !!response.data;
+            return service;
+        },
+    );
+    const isReady = services.some(value => !value);
+    if (isReady === true) {
```

Comment:
Please consider updating the name to `isNotReady` instead. `isReady` seems misleading considering it represents a failing endpoint.

```suggestion
    const isNotReady = services.some(value => !value);
    if (isNotReady) {
```